src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
I the interest of moving this along I changed all the ref callbacks to bound class methods. Let me know what you think.